src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -3,18 +3,33 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.Drawing;
 using System.Linq;
 using System.Reflection;
+using Res = Dynamo.Wpf.Properties.Resources;
 
 namespace Dynamo.ViewModels
 {
+    /// <summary>
+    /// The next enum will contain the posible values for Scaling (Visual Settings -> Geometry Scaling section)
+    /// </summary>
+    public enum GeometryScaleSize
+    {
+        Small,
+        Medium,
+        Large,
+        ExtraLarge
+    }
```

Comment:
Is there a similar enum for the previous UI that we will need to obsolete? Just curious

Reply:
@QilongTang I obsoleted the existing enum located in DynamoCoreWpf\UI\Prompts\ChangeScaleFactorPrompt.xaml.cs file, see commit: https://github.com/DynamoDS/Dynamo/pull/11636/commits/0fb11cde9886451ae02192f52dfc92552696ce67